cmd/hashsum/hashsum.go

```diff
@@ -55,6 +59,32 @@ func GetHashsumOutput(filename string) (out *os.File, close func(), err error) {
 	return out, close, nil
 }
 
+// HashStream produces hash from a stream
```

Comment:
This function should go in `operations/operations.go`

We can then write a unit test for it in there where we have lots of useful framework.

In general we try to do the absolute minimum in the cmd/ files and do everything in operations where it is easier to test.